tests/__init__.py

```diff
@@ -180,16 +182,17 @@ def login(self, user_id=TEST_USER_ID):
         Taking advantage of testing backdoor in views.auth.login()
 
         """
-        return self.client.get('/login/TESTING?user_id={0}'.format(user_id),
-                follow_redirects=True)
+        return self.client.get(
+            '/login/TESTING?user_id={0}'.format(user_id),
```

Comment:
Can you refactor to use `query_string`?
[See this example](https://github.com/uwcirg/true_nth_usa_portal/pull/2512#discussion_r204474339)